spec/features/issues_spec.rb

```diff
@@ -90,43 +90,45 @@
 
     let(:issue) { @issue }
 
-    it "should allow filtering by issues with no specified milestone" do
+    it 'should allow filtering by issues with no specified milestone' do
       visit project_issues_path(project, milestone_id: '0')
 
-      page.should_not have_content 'foobar'
-      page.should have_content 'barbaz'
-      page.should have_content 'gitlab'
+      expect(page).not_to have_content 'foobar'
+      expect(page).to have_content 'barbaz'
+      expect(page).to have_content 'gitlab'
     end
 
-    it "should allow filtering by a specified milestone" do
+    it 'should allow filtering by a specified milestone' do
       visit project_issues_path(project, milestone_id: issue.milestone.id)
 
-      page.should have_content 'foobar'
-      page.should_not have_content 'barbaz'
-      page.should_not have_content 'gitlab'
+      expect(page).to have_content 'foobar'
+      expect(page).not_to have_content 'barbaz'
+      expect(page).not_to have_content 'gitlab'
     end
 
-    it "should allow filtering by issues with no specified assignee" do
+    it 'should allow filtering by issues with no specified assignee' do
       visit project_issues_path(project, assignee_id: '0')
 
-      page.should have_content 'foobar'
-      page.should_not have_content 'barbaz'
-      page.should_not have_content 'gitlab'
+      expect(page).to have_content 'foobar'
+      expect(page).not_to have_content 'barbaz'
+      expect(page).not_to have_content 'gitlab'
     end
 
-    it "should allow filtering by a specified assignee" do
+    it 'should allow filtering by a specified assignee' do
       visit project_issues_path(project, assignee_id: @user.id)
 
-      page.should_not have_content 'foobar'
-      page.should have_content 'barbaz'
-      page.should have_content 'gitlab'
+      expect(page).not_to have_content 'foobar'
+      expect(page).to have_content 'barbaz'
+      expect(page).to have_content 'gitlab'
     end
   end
 
   describe 'filter issue' do
     titles = ['foo','bar','baz']
     titles.each_with_index do |title, index|
-      let!(title.to_sym) { create(:issue, title: title, project: project, created_at: Time.now - (index * 60)) }
+      let!(title.to_sym) { create(:issue, title: title,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.